db/1.9/modules/ROOT/pages/database-connector-query-single.adoc

```diff
@@ -0,0 +1,204 @@
+== Query a Single Record from a Database Example - Mule 4
+
+The *Single Query* operation enables you to execute an SQL query that returns a single record (row) from a database and access the result payload without any additional transformations. This operation does not apply streaming for any of the returned fields of the record. When the query is executed, all the values of the returned row are read into memory and returned in the payload. You must be careful, because if your row is too large you could run out of memory.
```

Comment:
I suggest: "This operation does not stream any of the returned fields."

Add a comma after "large."